src/main/java/Storage.java

```diff
@@ -1,19 +1,37 @@
+/**
+ * Storage acts to load and save the task list of the user.
+ * It contains methods to load upon the start of Duke and
+ * methods to save after every method call that changes
+ * the TaskList object.
+ * */
+
 import java.io.*;
 
 public class Storage {
 
     private String FILEPATH;
 
+    /**
+     * Default constructor. Takes in a String that is the filepath of the saved file.
+     * @param FILEPATH refers to the path of the saved file.
+     */
     public Storage (String FILEPATH) {
         this.FILEPATH = FILEPATH;
     }
 
-    public File load(){
+    /**
+     * Returns a File object with the given filepath.
+     * */
+    public File load() {
         File f = new File(FILEPATH);
         return f;
```

Comment:
I would suggest naming your variable "file" rather than "f" as it may cause confusion to yourself or others in the future. I myself have made the same mistake.